app/assets/javascripts/discourse/app/templates/components/sidebar/section-link.hbs

```diff
@@ -7,14 +7,39 @@
     @current-when={{@currentWhen}}
     @title={{@title}}
   >
+    {{#if @prefixValue }}
+      <span class="sidebar-section-link-prefix {{@prefixType}} {{@prefixCSSClass}}" style={{this.prefixCSS}}>
+        {{#if (eq @prefixType "image")}}
+          <img src={{@prefixValue}}>
```

Comment:
I think we should add CSS classes for these too so that it can easily be targeted with CSS.